serverless/knative-client.adoc

```diff
@@ -0,0 +1,37 @@
+[id="knative-client"]
+= Using Knative Client
+include::modules/common-attributes.adoc[]
+:context: knative-client
+
```

Comment:
Right here, you need a line that says "toc::[]", so that all of the included modules below show up as a TOC at the top of the assembly. (You should have blank lines above and below it, but just look at another assembly to see how it's done).